frontend/encore/simple-example.rst

```diff
@@ -180,8 +180,10 @@ The import and export Statements
 --------------------------------
 
 Instead of using ``require`` and ``module.exports`` like shown above, JavaScript
-has an alternate syntax, which is a more accepted standard. Choose whichever you
-want: they do the same thing.
+has an alternate syntax, which is the new finalized ECMAScript standard for using modules in the browser, including
```

Comment:
I propose the following reword to not say *"the new finalized"* (because that expression won't age well):

```rst
Instead of using ``require`` and ``module.exports`` like shown above, JavaScript
provides an alternate syntax based on the `ECMAScript 6 modules`_ that includes
the ability to use dynamic imports.
```

And at the bottom of the article, the URL for the "ECMAScript 6 modules" link must be added:

```rst
.. _`ECMAScript 6 modules`: https://hacks.mozilla.org/2015/08/es6-in-depth-modules/
```